searxng/searxng
# [fix] simple theme: make LESS sources available on non build hosts

## What does this PR do?

Include the LESS source files directly in the `.map` files [1].

[1] https://github.com/gruntjs/grunt-contrib-less#outputsourcefiles

## Why is this change important?

[fix] simple theme: make LESS sources available on non build hosts

## How to test this PR locally?

    make node.clean run

In the browser open debug mode and see if you have access to the LESS source of e.g. `normalize.css`

    /static/themes/simple/node_modules/normalize.css/normalize.css

## Author's checklist

Drawback: `.map` files grow up in size, but this should not bother, because the files are only loaded in debug mode.

## Related issues

Suggested-by: @dalf https://github.com/searxng/searxng/pull/575#issuecomment-984058868
